<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1211</width>
    <height>976</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton[is_grid=true]{
    width: 60px;
    height: 60px;
    margin: 0px 0px;
}
QFrame#GridFrame{
	background-color: #dedede
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="PageControler">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>1011</width>
      <height>811</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="StartPage">
     <widget class="QLabel" name="StartLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>190</y>
        <width>761</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe Arabic</family>
        <pointsize>72</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Mine Clearance</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>460</y>
        <width>221</width>
        <height>211</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="GameButton">
         <property name="font">
          <font>
           <family>Adobe Arabic</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>ÂºÄÂßãÊ∏∏Êàè</string>
         </property>
         <property name="text">
          <string>Game</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="AdjustBuuton">
         <property name="font">
          <font>
           <family>Adobe Arabic</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Ê∏∏ÊàèËÆæÁΩÆ</string>
         </property>
         <property name="text">
          <string>Adjust</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="DevButton">
         <property name="font">
          <font>
           <family>Adobe Arabic</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>ÂºÄÂèëËÄÖÈÄâÈ°π</string>
         </property>
         <property name="text">
          <string>Developer</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="GamePage">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>40</y>
        <width>114</width>
        <height>123</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="MarkMineButton">
         <property name="toolTip">
          <string>Ê†áËÆ∞Âú∞Èõ∑</string>
         </property>
         <property name="text">
          <string>üõé</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="MarkGridButton">
         <property name="toolTip">
          <string>Ê†áËÆ∞Á©∫Âú∞</string>
         </property>
         <property name="text">
          <string>‚ñë‚ñë</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="RestartButton">
         <property name="toolTip">
          <string>ÂÜçÊù•‰∏ÄÊ¨°</string>
         </property>
         <property name="text">
          <string>‚Üª</string>
         </property>
         <property name="as" stdset="0">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="GridFrame">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>751</width>
        <height>611</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="AdjustPage"/>
    <widget class="QWidget" name="DevPage"/>
   </widget>
   <widget class="QFrame" name="StatusBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>1061</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="PageStatus">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>20</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe Arabic</family>
       <pointsize>20</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>start page</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="GotoStart">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>ËøîÂõû</string>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="GameStatus">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>621</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe Arabic</family>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Prepareing</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>GameButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>goto_GamePage_event()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>566</x>
     <y>579</y>
    </hint>
    <hint type="destinationlabel">
     <x>1038</x>
     <y>847</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AdjustBuuton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>goto_AdjustPage_event()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>499</x>
     <y>676</y>
    </hint>
    <hint type="destinationlabel">
     <x>987</x>
     <y>874</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DevButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>goto_DevPage_event()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>451</x>
     <y>771</y>
    </hint>
    <hint type="destinationlabel">
     <x>944</x>
     <y>906</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>GotoStart</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>goto_StartPage_event()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>72</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MarkMineButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>goto_MarkMineButton_event()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>994</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>1148</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MarkGridButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>goto_MarkGridButton_event()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>967</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>1181</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RestartButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>goto_RestartButton_event()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>995</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>1171</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>goto_GamePage_event()</slot>
  <slot>goto_AdjustPage_event()</slot>
  <slot>goto_DevPage_event()</slot>
  <slot>goto_StartPage_event()</slot>
  <slot>goto_MarkMineButton_event()</slot>
  <slot>goto_MarkGridButton_event()</slot>
  <slot>goto_RestartButton_event()</slot>
 </slots>
</ui>
